src/app/components/header/header.component.html

```diff
@@ -8,31 +8,31 @@ <h1 class="title" routerLink="/">Verseghy</h1>
       <i class="material-icons menu-close" (click)="drawer = !drawer">close</i>
     </div>
     <div class="menu">
-      <a href="" routerLink="/" class="active">Főoldal</a>
+      <a href="" routerLink="/" class="active" (click)="drawer = !drawer">Főoldal</a>
     </div>
-    <div class="menu" (click)="submenu1 = !submenu1" [ngClass]="{'open' : submenu1}">
-      <span>Menza</span>
+    <div class="menu" (click)="submenu1 = !submenu1;" [ngClass]="{'open' : submenu1}">
```

Comment:
Why did you add this semicolon here? For a single line of code, we don't use semicolons